cdap-docs/application-templates/source/etl/templates/sinks/real-time/cube.rst

```diff
@@ -8,17 +8,19 @@ Sinks: Real-time: Cube
 
 .. rubric:: Description
 
-Real-time Sink that writes data to a Cube dataset
+Real-time Sink that writes data to a Cube dataset.
 
-This RealtimeCubeSink takes a StructuredRecord in, maps it to a CubeFact, and writes it to
-a Cube dataset identified by the Dataset Name property.
+RealtimeCubeSink takes a StructuredRecord in, maps it to a CubeFact, and writes it to
+the Cube dataset identified by the name property.
 
 If Cube dataset does not exist, it will be created using properties provided with this
 sink.
 
 .. rubric:: Use Case
 
-TODO: Fill me out
+A RealtimeCubeSink is used to write data into a Cube from data sources to allow further OLAP data analysis.
+E.g. with RealtimeCubeSink you can write data from a Stream into a Cube as it arrives to perform complex
```

Comment:
For example, you can write data from a Stream into a Cube as the data arrives to perform complex